packages/slice-machine/lib/builders/CustomTypeBuilder/Layout/Header.tsx

```diff
@@ -77,7 +76,7 @@ const CustomTypeHeader = ({
     <Header
       MainBreadcrumb={
         <Fragment>
-          <FiLayout /> <Text ml={2}>Custom Types</Text>
+          <MdSpaceDashboard /> <Text ml={2}>Custom Types</Text>
```

Comment:
I prefer when the two icons are the same,  but on the figma, the icons seems to be different between navigation and this page. (from what I saw on the ticket)

Reply:
The figma preview on the ticket is not up to date, all the icon for custom type list page, navigation and custom type detail page are the same.